### DISCUSSION TITLE
Problem with updating

### DISCUSSION THREAD

**Discussion by jakudo** [RavenSystem/esp-homekit-devices#1645] [General]
Hi,

lately I had several issues with updating my Sonoff devices (mini and SV).The update stuck in some kind of loop. I can see that it is connected, pingable and it downloading some data, however, after about 2 minutes, it reconnect and does that again. I had this problem with 3 Sonoff mini and one SV. Update was from some older fw (7 or 8 ) and from 9.3 -> 9.4.
After I reflashed them manually, it works ok.
Any idea what might be wrong?

**Comment by RavenSystem** [comment]
Logs?

**Reply by avia07** [reply]
I have the same problem with esp8266.   I can not update with any bootloaders.

**Reply by jakudo** [reply]
Sorry, I don't know how to get the log. After I click update, the HAA Manager shows
Reachable - No
Blocked - No
HomeKit Server - Unreachable
Setup Mode - Unreachable

There is some progress - the sonoff SV got updated and operational again after several hours (I didn't reflash it since it is outside next to my gate and I didn't have time to do that yesterday nigh). It took at least 2 hours to update (after that I didn't check it and checked that this morning). The WiFi signal is very good. Usually the update took about 2-3 minutes.

**Reply by Alex-us425** [reply]
привет! проблема есть ! позавчера вечером что то произошло и все не обновленные устройства стали обновляться как надо!
на следующее утро лафа закончилась и все перестало работать! Не настраиваются ка вновь прошитые устройства так и не выполняется обновление старых!

**Reply by avia07** [reply]
Победил проблему весьма нестандартным способом. После прошивки загрузчика подключил к хот споту смартфона и через сотовую сеть (LTE) пошло обновление и все прошилось как надо.  Сегодня так же попробовал через смартфон - все работает. Не знаю почему, но через интернет провайдера ничего не грузилось. 
I solved the problem in a very non-standard way. After flashing the bootloader, I connected the ESP to the hot spot of the smartphone and an update went through the cellular network (LTE) and everything was flashed as it should. Today I also updated it via LTE - everything works. I don’t know why, but nothing was loaded through the wired Internet.

**Reply by jakudo** [reply]
that is an interesting solution. However, in case you have some Sonoffs already installed somewhere and just want to update them, it is not possible to reconnect all of them to LTE.

**Reply by Alex-us425** [reply]
Пробовал и через телефон и через роутер с сим картой от мобильного оператора мегафон! Изменений никаких🥲

**Reply by avia07** [reply]
Попробуйте залить загрузчик от версии 7.2.1

**Reply by Alex-us425** [reply]
У меня загрузчик 5.5.2

**Reply by Alex-us425** [reply]
Ок попробую более новый если он уже есть

**Reply by avia07** [reply]
Позавчера и вчера у меня с ним получилось.   
Можно попробовать поднять свой сервер и с него  обновиться.  Правда у меня на Маке не получилось. Сервер работает, доступ есть, через браузер  все хорошо, но по логам  обращение от soonoff к серверу есть, а обновление не грузится. ХЗ почему.

**Reply by avia07** [reply]
https://github.com/RavenSystem/haabeta/releases

**Reply by avia07** [reply]
Нет не эти
Это Бета

**Reply by avia07** [reply]
Отсюда
https://github.com/RavenSystem/haa/releases

**Reply by avia07** [reply]
7.2.1

**Reply by avia07** [reply]
> that is an interesting solution. However, in case you have some Sonoffs already installed somewhere and just want to update them, it is not possible to reconnect all of them to LTE.

Another. Flash 7.2.1 bootloader. Then AutoOTA.

**Reply by Alex-us425** [reply]
прошиваем только fullhaaboot.bin ?

**Reply by VaBanck** [reply]
Да

**Reply by VaBanck** [reply]
Зайдите в discord там быстрее все будет решаться

**Reply by VaBanck** [reply]
https://discord.gg/v8hyxj2
